src/components/organism/Modal/Profile/SocialProfileEditForm.tsx

```diff
@@ -67,9 +70,12 @@ export const SocialProfileEditForm: FC<Props> = ({ did }) => {
   const onClickSubmit = async (data: OrbisProfileDetail, e?: BaseSyntheticEvent) => {
     e?.preventDefault()
     e?.stopPropagation()
+    if (!icon) console.error("NO pfp")
```

Comment:
for this one, you probably need to create the content property directly and pass it to update() function instead of setValue.
https://user-images.githubusercontent.com/46702100/251142260-1c48ca88-d6b1-4979-8228-6f7899c04e68.png